src/Microsoft.AspNetCore.Razor.Language/MemoryRazorProject.cs

```diff
@@ -0,0 +1,201 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.AspNetCore.Razor.Language
+{
+    public class MemoryRazorProject : RazorProject
```

Comment:
VirualProjectFileSystem

Reply:
`internal` also think this should probably be in the Razor.Tools project.